src/modules/core/components/Permission/Permission.css

```diff
@@ -0,0 +1,26 @@
+.wrapper {
```

Comment:
I think the labels should have some small `margin-right` value applied to them. Something like:
```css
margin-right: 4px;
```

If you really are in the mood, you can add an `Apperance` prop to this, which just have the margin option:
```js
interface Appearance {
  margin: 'small' | 'none';
}
```

You can look up an example of how to set this up in one of the other core components

Reply:
Being super honest, i think it shouldn't be permissions responsibility, it should be wrapper component responsibility - so in case when we display permissions as a list - list will make sure that children have correct space between. If we display them alone - they just don't have